docs/guides/user-guide/general-faqs.md

```diff
@@ -0,0 +1,189 @@
+
+### Can I change the Node version in my project?
+
+The current state of Codeflow does not support users being able to set their own Node version.
+
+### Why can't I see my terminal?
+
+Please note that the terminal is only available for WebContainer-based projects. You can use the keyboard binding `^` `` ` `` (the caret and the back quote keys) to toggle the terminal.
+
+### Can I run unit tests in StackBlitz?
+
+You can use [Jasmine](https://jasmine.github.io/), a unit testing tool for JavaScript.
+
+It is also possible to run Angular unit tests inside WebContainer-based projects, but not Engineblock-based projects. The easiest way to start is by navigating to node.new, which creates a WebContainer-based project automatically!
+
+### Can I develop ElectronJS apps on StackBlitz?
+
+Since Electron is a native app, this cannot be supported since StackBlitz runs entirely within the browser.
+
+## Codeflow Editor
+
+### What is the Codeflow Editor?
+
+The Codeflow Editor utilizes our Codeflow technology to bring VSC code into the browser. This editor is capable of running full development workflows, including pushing commits and creating pull requests to GitHub. This bypasses the need to clone a repository or install dependencies on your local machine. If you haven't already, you can check out Codeflow [here](https://stackblitz.com/codeflow)!
+
+### Can I use Vim bindings?
+
+Our Codeflow editor is built on top of VS Code so you can enable the VIM extension:
+
+1. Open a github repository in Codeflow (like this: pr.new/vitejs/vite)
+2. Navigate to the "Extensions" pane
+3. Search for "Vim"
+4. Click on the VIM extension and then click the "enable" button.
+
+## Web Publisher
+
+### What is Web Publisher?
+
+Web Publisher is a single file editor and publishing tools that can be used for smaller or text-based changes such as editiong documentation or blogs. Web Publisher uses Codeflow technology that allows users to create, push, and publish changes all in their browser. The GitHub integration creates a seamless workflow that's fit for developers and non-coding folks alike!
+
+This environment is ideal for:
+- No-tech or low-tech users
+- Text or formatting changes in documentation or blogs
+- Quick fixes that don't require dependency or compilation changes
+
+You can learn more about Web Publisher [here](https://developer.stackblitz.com/codeflow/content-updates-with-web-publisher).
+
+## Classic Editor SDK
+
+### What is the Classic Editor SDK?
+
+Our SDK (software development kit) allows users to create StackBlitz projects to be opened in a new window or embedded in your docs, example pages, or blog posts. Projects are controlled by using the SDK's VM (virtual machine) interface. The VM can be used to control the UI of an embedded StackBlitz project, change the currently open file(s), and read and write files frmo the project's virtual filesystem.
```

Comment:
Our JavaScript-based SDK..... allows users to created SB Classic Editor projects.

Remove VM info. "The SDK can be used to control the UI......"